tickets/templates/tickets/barbu_base.html

```diff
@@ -21,7 +21,6 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```

Comment:
You can view this file's changes with "ignore whitespace" setting:
https://github.com/fuegoaustral/ticketera/pull/115/files?w=1

Reply:
buen truco! no lo conocía <3